syntax = "proto3";

package saq.sandwich.proto.api.v1;


// Certificate and public key encoding format.
enum ASN1EncodingFormat {
  // PEM format.
  ENCODING_FORMAT_PEM = 0;
  // DER format.
  ENCODING_FORMAT_DER = 1;
}
